CHANGELOG.md

```diff
@@ -9,6 +9,10 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 
 [6.0.0-rc2]: https://github.com/microsoft/CCF/releases/tag/6.0.0-rc2
 
+### Added
+
+- Attestations of new SNP nodes must be from a trusted TCB version higher than the minimum TCB version stored for that CPU model in `public:ccf.gov.nodes.snp.tcb_versions`. Added `add_snp_tcb_version(cpuid, tcb_version)` and `remove_snp_tcb_version(cpuid)` governance actions. New clusters will automatically populate the TCB version, pre-existing clusters must be set a TCB version when upgrading. (#6837)
+
```

Comment:
It sounds like those should be `set_` and `remove_` rather than `add_`, `add_` is used for sets, and here this is only a single min TCB version per CPUID, right?

We don't have this in the doc, but in practice we follow this set of guidelines for governance actions: https://github.com/microsoft/CCF/discussions/2169#discussioncomment-373490